tmVhdlProducer/vhdlhelper.py

```diff
@@ -251,6 +255,30 @@ def bx_encode_4_array(value: int) -> str:
     """
     return format([2, 1, 0, -1, -2].index(value), 'd')
 
+def get_files_hash_value(dir_path):
+    """Calculate hash value of the content of all .py and .vhd files in 'dir_path'.
+    """
+    py_files = dir_path+"/**/*.py"
+    vhd_files = dir_path+"/**/*.vhd"
+
+    x = hashlib.sha256()
```

Comment:
`h = hash.sha256()` or `sha256_hash = hash.sha256()` would increase code readability (note `hash` is a built in function, so try to avoid `hash = hash.sha256()`).

Reply:
I do not understand! Why is "h" better than "x"?

Reply:
`h` for hash. While `x` is a perfectly valid variable name, it is quite generic and provides no hints about the variable's intended use. In contrast, `h` or `sha256_hash` offers immediate context. See also the Zen of Python's "Explicit is better than implicit" principle (PEP 20), https://peps.python.org/pep-0020/